vm/qemu/qemu.go

```diff
@@ -360,6 +362,9 @@ func (inst *instance) Boot() error {
 			"-append", strings.Join(cmdline, " "),
 		)
 	}
+	if inst.cfg.Snapshot {
```

Comment:
This does not look equivalent to the current code. Current code adds snapshot only if inst.image != "" and inst.image != "9p".